CMakeLists.txt

```diff
@@ -22,15 +24,33 @@ set(tinyobjloader-examples-objsticher
 	${TINYOBJLOADEREXAMPLES_DIR}/obj_sticher/obj_sticher.cc
 	)
 
+#Install destinations
+set(TINYOBJLOADER_CMAKE_DIR lib/cmake)
+set(TINYOBJLOADER_INCLUDE_DIR include)
+set(TINYOBJLOADER_LIBRARY_DIR lib)
+set(TINYOBJLOADER_PKGCONFIG_DIR lib/pkgconfig)
+
 option(TINYOBJLOADER_BUILD_TEST_LOADER "Build Example Loader Application" OFF)
 option(TINYOBJLOADER_COMPILATION_SHARED "Build as shared library" OFF)
 
 if (TINYOBJLOADER_COMPILATION_SHARED)
 	add_library(tinyobjloader SHARED ${tinyobjloader-Source})
+	set_target_properties(tinyobjloader PROPERTIES
+		SOVERSION ${TINYOBJLOADER_SOVERSION}
+	)
 else()
 	add_library(tinyobjloader STATIC ${tinyobjloader-Source})
 endif()
 
+set_target_properties(tinyobjloader PROPERTIES VERSION ${TINYOBJLOADER_VERSION})
+
+target_include_directories(tinyobjloader INTERFACE
+	$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>
+	$<INSTALL_INTERFACE:${TINYOBJLOADER_INCLUDE_DIR}>
```

Comment:
FYI, I believe `install(...PUBLIC_HEADER DESTINATION...)` makes this unnecessary (and will better handle the appropriate absolute path, anyway)